api/observability/v1/filter_types.go

```diff
@@ -16,12 +16,12 @@ package v1
 
 // FilterType specifies the type of filter used in a pipeline
 //
-// +kubebuilder:validation:Enum:=addLabels;detectMultiline;drop;kubeAPIAudit;parse;prune
+// +kubebuilder:validation:Enum:=openShiftLabels;detectMultilineException;drop;kubeAPIAudit;parse;prune
 type FilterType string
 
 // Filter type constants, must match JSON tags of FilterTypeSpec fields.
 const (
-	FilterTypeAddLabels       FilterType = "addLabels"
+	FilterTypeOpenshiftLabels FilterType = "openShiftLabels"
 	FilterTypeDetectMultiline FilterType = "detectMultiline"
```

Comment:
Should here be detectMultiline**Exception** ?

Reply:
Agree.  I have this as part of a commit that has not been pushed yet

Reply:
Commit was on this branch.. updated here